README.md

```diff
@@ -433,13 +433,14 @@ override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
 ## Strings
 
 ```sh
-swiftgen strings -t structured-swift3 /path/to/Localizable.strings
+swiftgen strings -t structured-swift3 (/path/to/Localizable.strings | /path/to/dir)
```

Comment:
I think that notation might be confusing to people not familiar with it, risking to make some people actually type the pipe verbatim.

Instead I think it's clearer to split that in 2 different example command, like:

```sh
swiftgen strings -t structured-swift3 /path/to/Localizable.strings
# OR
swiftgen strings -t structured-swift4 /path/to/SomeLang.lproj
```

Reply:
sure thing